pkg/forwarder/vtunnel.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright © 2022 SUSE LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package forwarder implements a forwarding mechanism to forward
+// port mappings over Vtunnel.
+package forwarder
+
+import (
+	"encoding/json"
+	"net"
+
+	"github.com/rancher-sandbox/rancher-desktop-agent/pkg/types"
+)
+
+const vtunnelPeerAddr = "127.0.0.1:3040"
```

Comment:
We'll need to be able to thread this in from a command line argument… At least the port.